### PR TITLE
Add support for gitlab

### PR DESCRIPTION
**Pull request by ii8**

### CODE REVIEW

**Comment by SeanTAllen** [comment]
This looks good to me. @jemc any comments before I merge?

**Review by SeanTAllen — APPROVED** [review]

**Comment by SeanTAllen** [comment]
Thanks @ii8. Nice first contribution to stable.

**Review by srenatus — APPROVED** [review]
LGTM 😃

**Comment by srenatus** [comment]
We'll have to get #57 and #67 updated when this is in, but that's fine 👍

File: stable/dep.pony [hunk 1]
```diff
@@ -9,19 +9,23 @@ interface DepAny
...
+class DepGitHosted
+  let domain: String
   let bundle: Bundle box
   let info: JsonObject box
   let repo: String
   let subdir: String
   let git_tag: (String | None)
-  new create(b: Bundle box, i: JsonObject box) ? =>
+
+  new create(d: String, b: Bundle box, i: JsonObject box) ? =>
```
Comment (jemc) [inline stable/dep.pony]: Small change, but I'd prefer the parameter signature to put the new parameter at the end:
```pony
create(b: Bundle box, i: JsonObject box, d: String)
```

This keeps consistency a bit better with the other `Dep` types, which all take `b: Bundle box, i: JsonObject box` as their first two arguments.

**Review by jemc — CHANGES_REQUESTED** [review]

**Comment by mfelsche** [comment]
@jemc seems your issue has been resolved. we cool to mege now?

**Review by jemc — APPROVED** [review]

**Pull request merged**